### PR TITLE
Add missing attribute into document

### PR DESCRIPTION
**Pull request by ParthBrahmbhatt**
## This PR is a:

- [ ] Content fix or rewrite

## Summary

When this pull request is merged, it will add the missing attribute into the document which are already available into codebase.

## Additional information
Magento 2.3 :
https://github.com/magento/graphql-ce/blob/4a28abb62b4f879794b44c2b2230db15ddea46ba/app/code/Magento/QuoteGraphQl/etc/schema.graphqls#L152
https://github.com/magento/graphql-ce/blob/4a28abb62b4f879794b44c2b2230db15ddea46ba/app/code/Magento/QuoteGraphQl/etc/schema.graphqls#L153

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/graphql/reference/quote.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by erikmarr** [comment]
Assigning to @keharper so he can process all 2.3.2 query updates at the same time.

**Comment by osrecio** [comment]
@keharper Are You working on 2.3.2 branch right?

**Comment by keharper** [comment]
@osrecio, yes. The `quote` endpoint is undergoing major revisions for 2.3.2, and as of yesterday, it's still subject to change. I merged a couple of 2.3.2 PRs into the `develop` branch, but they weren't about `quote`. Hopefully, I can merge this and a few others by early next week.

File: guides/v2.3/graphql/reference/quote.md [hunk 1]
```diff
@@ -24,11 +24,13 @@ The `Cart` object can contain the following attributes:
 
 Attribute |  Data Type | Description
 --- | --- | ---
-`applied_coupon` | code | Contains the coupon code if used 
+`applied_coupon` | code | Contains the coupon code if used
```
Comment (keharper) [inline guides/v2.3/graphql/reference/quote.md]: The data type is now `AppliedCoupon`. Also update the description to say something like "The `AppliedCoupon` object contains the `code` text attribute, which provides the coupon code.

File: guides/v2.3/graphql/reference/quote.md [hunk 2]
```diff
@@ -24,11 +24,13 @@ The `Cart` object can contain the following attributes:
...
+`available_payment_methods` | AvailablePaymentMethod | Available payment methods
 `billing_address` | [CartAddress](#cartAddressAttributes) | Contains the billing address specified in the customer's cart
 `cart_id` | String | The unique ID that identifies the customer's cart
```
Comment (keharper) [inline guides/v2.3/graphql/reference/quote.md]: Delete this attribute.

File: guides/v2.3/graphql/reference/quote.md [hunk 3]
```diff
@@ -24,11 +24,13 @@ The `Cart` object can contain the following attributes:
 
 Attribute |  Data Type | Description
 --- | --- | ---
-`applied_coupon` | code | Contains the coupon code if used 
+`applied_coupon` | code | Contains the coupon code if used
+`available_payment_methods` | AvailablePaymentMethod | Available payment methods
```
Comment (keharper) [inline guides/v2.3/graphql/reference/quote.md]: Add a new table for `AvailablePaymentMethod`.

File: guides/v2.3/graphql/reference/quote.md (hunk 2 above)
Comment (keharper) [reply guides/v2.3/graphql/reference/quote.md]: Add the `email` attribute

File: guides/v2.3/graphql/reference/quote.md [hunk 4]
```diff
@@ -24,11 +24,13 @@ The `Cart` object can contain the following attributes:
...
 `billing_address` | [CartAddress](#cartAddressAttributes) | Contains the billing address specified in the customer's cart
 `cart_id` | String | The unique ID that identifies the customer's cart
 `items` | [CartItemInterface](#cartItemsInterface) | Contains the items in the customer's cart
 `shipping_addresses` | [CartAddress](#cartAddressAttributes) | Contains one or more shipping addresses
+`selected_payment_method` | SelectedPaymentMethod | Selected payment method
```
Comment (keharper) [inline guides/v2.3/graphql/reference/quote.md]: This item is not in alphabetical order. Move it up one.

Add a section for `SelectedPaymentMethod`.

**Comment by keharper** [comment]
@ParthBrahmbhatt There have been some code changes since you opened this PR. If you can't see these changes externally, wait a few days until the latest changes have been merged.

**Comment by keharper** [comment]
Merging so that I can go ahead with my plan to split this topic into multiple topics.

**Pull request merged**